centreon/www/front_src/src/ResourceAccessManagement/Listing/index.tsx

```diff
@@ -0,0 +1,3 @@
+import ResourceAccessRulesListing from './Listing';
+
+export default ResourceAccessRulesListing;
```

Comment:
We use index.ts files in to export named component like the following
```suggestion
export { default as ResourceAccessRulesListing } from './Listing';
```

Reply:
Done